### PR TITLE
replaced stack trace prints with log entries

### PR DESCRIPTION
**Pull request by antares1470**
### Description of the Change

Replaced instances of Throwable.printStackTrace(...) with logger.log(...) as suggested by sonar

### CODE REVIEW

File: nbproject/platform.properties [hunk 1]
```diff
@@ -3,7 +3,6 @@ cluster.path=\
     ${nbplatform.active.dir}/harness:\
     ${nbplatform.active.dir}/platform
 disabled.modules=\
-    org.jdesktop.layout,\
```
Comment (arcturus2) [inline nbproject/platform.properties]: Was this by accident?

File: CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/state/TableStateIOProvider.java [hunk 2]
```diff
@@ -68,7 +68,6 @@ public void readObject(final int attributeId, final int elementId, final JsonNod
                     state.columns.add(cs);
                 } catch (IllegalArgumentException ex) {
                     LOGGER.log(Level.SEVERE, "Segment value {0} illegal", segmentText);
```
Comment (cygnus-x-1) [inline CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/state/TableStateIOProvider.java]: can you pass ```ex``` through to the logger too?

File: nbproject/platform.properties (hunk 1 above)
Comment (cygnus-x-1) [reply nbproject/platform.properties]: This happened in constellation-adaptors when i changed it from an 'application' to a 'module collection' - most likely unintended here.

Comment (antares1470) [reply nbproject/platform.properties]: Can confirm was unintended

File: CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/state/TableStateIOProvider.java (hunk 2 above)
Comment (antares1470) [reply CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/state/TableStateIOProvider.java]: Change has now been made

**Comment by arcturus2** [comment]
@antares1470  just letting you know there is a compile failure here:
```
app/constellation/CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/state/TableStateIOProvider.java:70: error: no suitable method found for log(Level,String,String,IllegalArgumentException)
 [nb-javac]                     LOGGER.log(Level.SEVERE, "Segment value {0} illegal", segmentText, ex);
 [nb-javac]                           ^
```

**Comment by antares1470** [comment]
> 
> 
> Please fix the compile failure mentioned [#307 (comment)](https://github.com/constellation-app/constellation/pull/307#issuecomment-601470396) thanks!

Thanks @arcturus2 that has now been fixed. Apparently netbeans was a bit slow to tell me of the issue, wouldn't have pushed it otherwise :)

**Comment by arcturus2** [comment]
All good @antares1470, figured that would be the case.

**Pull request merged**